.vscode/settings.json

```
@@ -1,5 +1,10 @@
 {
     "yaml.schemas": {
         "./defaults.schema.json": ["/defaults*.yml"]
-    }
+    },
+    "[yaml]": {
+        "editor.rulers": [
+            130
+        ]
+    },
```

This probably is the reason it breaks
```suggestion
    }
```

---

you're right. I didn't realize the action uses this file directly. It [parses it as normal JSON](https://github.com/nwisbeta/validate-yaml-schema/blob/d6d0e9a1ff5fceb6f59b9751c79f899b324730fc/src/main.ts#L25) but this is technically jsonc